specification/paloaltonetworks/resource-manager/readme.typescript.md

```diff
@@ -24,4 +24,19 @@ directive:
           }
         }
       }
+  - from: PaloAltoNetworks.Cloudngfw.json
```

Comment:
@josefree will this require ARM review? this change only happens in typescript scope for client generation.